zanata-common-api/src/main/java/org/zanata/rest/dto/stats/contribution/BaseContributionStatistic.java

```diff
@@ -10,25 +13,27 @@
  * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
+@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+@JsonPropertyOrder({ "approved", "rejected", "translated", "needReview" })
 public class BaseContributionStatistic implements Serializable {
 
-    private int approved;
-    private int needReview;
-    private int translated;
-    private int rejected;
+    private Integer approved;
+    private Integer needReview;
+    private Integer translated;
+    private Integer rejected;
```

Comment:
Lets add a comment about why these are `Integer`s and not `int`s. Since the API is sound, we can change the implementation later if we want to.